packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx

```diff
@@ -42,22 +42,27 @@ const CreateDaoProgress: React.FC<{
   const [showAction, setShowAction] = useState<'none' | 'fail' | 'register' | 'finish'>('none');
   const [rule, SetRule] = useState<BytesLike>('');
```

Comment:
BTW, while you're on this file, can you also change line 33 to use ether's constants.AddressZero instead of defining ADDRESS_ZERO in this file?

https://docs.ethers.io/v5/api/utils/constants/#constants-AddressZero

Reply:
perfect, thanks